.github/workflows/visual-regression-tests.yml

```
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  visual-regression-test-components:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true # Also runs `bundle install`
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+This conversation was marked as resolved by injms
+- name: Get Yarn cache directory path
+  id: yarn-cache-directory-path
+  run: echo "::set-output name=dir::$(yarn cache dir)"
+- name: Check for cached Yarn dependencies
+  uses: actions/cache@v2
+  with:
+    path: ${{ steps.yarn-cache-directory-path.outputs.dir }}
+    key: yarn-${{ hashFiles('**/yarn.lock') }}
+    restore-keys: yarn-
```

If we update the `setup-node` image to version 2, we can replace this slightly odd caching stuff with the built in caching available:

```suggestion 
- uses: actions/setup-node@v2.5.1
   with:
     node-version: "12"
     cache: yarn
```